rpm/nemo-qml-plugin-dbus-qt5.spec

```diff
@@ -20,25 +23,46 @@ Requires:   %{name} = %{version}-%{release}
 Requires:   qt5-qtdeclarative-import-qttest
 Requires:   qt5-qtdeclarative-devel-tools
 
+%package testd
```

Comment:
Is there any reason to have this in its own package? I suggest that you combine testd and the existing tests package. It is okay to have autotests and manual tests in the same package.